src/ZoneServer/Skills/Handlers/Swordsman/Peltasta/SwashBuckling.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using Melia.Shared.L10N;
+using Melia.Shared.Game.Const;
+using Melia.Shared.World;
+using Melia.Zone.Network;
+using Melia.Zone.Skills.Handlers.Base;
+using Melia.Zone.World.Actors;
+using Melia.Zone.World.Actors.CombatEntities.Components;
+using Melia.Shared.Data.Database;
+using Melia.Zone.Scripting.AI;
+
+namespace Melia.Zone.Skills.Handlers.Swordsman.Peltasta
+{
+	/// <summary>
+	/// Handler for the Peltasta skill Swash Buckling
+	/// </summary>
+	[SkillHandler(SkillId.Peltasta_SwashBuckling)]
+	public class Peltasta_SwashBuckling : IGroundSkillHandler
+	{
+		/// <summary>
+		/// Handles the skill, giving the caster a buff and debuffing enemies in target area.
+		/// </summary>
+		/// <param name="skill"></param>
+		/// <param name="caster"></param>
+		/// <param name="originPos"></param>
+		/// <param name="farPos"></param>
+		/// <param name="target"></param>
+		public void Handle(Skill skill, ICombatEntity caster, Position originPos, Position farPos, ICombatEntity designatedTarget)
+		{
+			var duration = TimeSpan.FromSeconds(5 + 2 * skill.Level);
+
+			if (!caster.TrySpendSp(skill))
+			{
+				caster.ServerMessage(Localization.Get("Not enough SP."));
+				return;
+			}
+			skill.IncreaseOverheat();
+			caster.SetAttackState(true);
+
+			Send.ZC_SKILL_MELEE_GROUND(caster, skill, caster.Position, null);
+
+			// This buff is mentioned in the skill file, but it seems like it was tied to abilities that have been removed
```

Comment:
I would argue that we should just add the respective checks and buff applications instead of only commenting about them. If the ability doesn't appear in the wild anymore that's fine, but if someone wanted to add it, it will immediately work, which is always nice.

Reply:
I would probably have done so, but at the time I was not sure what these buffs did or what ability applied them.  When I go back to add the rest of Peltasta's other skills I'll probably try to find that information, I've been adding it to my more recent skill PRs.